gen/calc.py

```diff
@@ -287,14 +287,17 @@ def calculate_config_id(dcos_image_commit, user_arguments, template_filenames):
         "template_filenames": json.loads(template_filenames)})
 
 
-def calculate_cluster_packages(package_names, config_id):
-    def get_package_id(package_name):
-        pkg_id_str = "{}--setup_{}".format(package_name, config_id)
+def calculate_config_package_ids(config_package_names, config_id):
+    def get_config_package_id(config_package_name):
+        pkg_id_str = "{}--setup_{}".format(config_package_name, config_id)
         # validate the pkg_id_str generated is a valid PackageId
         return pkg_id_str
 
-    cluster_package_ids = list(sorted(map(get_package_id, json.loads(package_names))))
-    return json.dumps(cluster_package_ids)
+    return json.dumps(list(sorted(map(get_config_package_id, json.loads(config_package_names)))))
+
+
+def calculate_cluster_packages(config_package_ids, package_ids):
```

Comment:
We need an easy way to access the config packages ids for rendering and generating additional artifacts to upload.

https://github.com/dcos/dcos/pull/524/files#diff-aab0f8dbae82b6ad54ff55e6f2195685R780
https://github.com/dcos/dcos/pull/524/files#diff-4dbd5bc0ea362b783a3e0273771660a9R234

I could've instead added a helper funciton that loops through the cluster packages and returns packages with versions starting with `setup_`, but that seemed less straightforward.